src/db.rs

```diff
@@ -414,6 +423,9 @@ impl DbInner {
 
     fn freeze_memtable(&self) -> Result<(), SlateDBError> {
         let mut guard = self.state.write();
+        if guard.memtable().is_empty() {
+            return Ok(());
+        }
```

Comment:
I added this change to avoid flushing empty memtables on close/flush. Doing that would cause us to flush a memtable with no "latest tick" because no data was written. (Doing so would cause a hard failure)

cc @hachikuji